GliaWidgetsTests/SecureConversations/ChatTranscript/TranscriptModel.Environment.Failing.swift

```diff
@@ -73,6 +73,10 @@ extension SecureConversations.TranscriptModel.Environment {
             fail("\(Self.self).createSendMessagePayload")
             return .mock()
         },
-        log: .failing
+        log: .failing,
+        maximumUploads: {
+            fail("\(Self.self).maximumUploads")
+            return 2
```

Comment:
Was 2 chosen in these mocks and failings for a reason?

Reply:
No reasons

Reply:
i suppose this is failing mock, so in fact the return value may be random, because unit test will be failed anyway.